cli-commands/compiler/etherlime-contract-schema/index.js

```diff
@@ -150,20 +150,21 @@ function chain() {
 }
 
 var EtherlimeContractSchema = {
-  validate: function (contractObj) {
-    var ajv = new Ajv({ useDefaults: true });
-    ajv.addSchema(abiSchema);
-    ajv.addSchema(networkObjectSchema);
-    ajv.addSchema(contractObjectSchema);
-    if (ajv.validate("contract-object.spec.json", contractObj)) {
-      return contractObj;
-    } else {
-      throw ajv.errors;
-    }
-  },
+  // validate: function (contractObj) {
+  //   var ajv = new Ajv({ useDefaults: true });
+  //   ajv.addSchema(abiSchema);
+  //   ajv.addSchema(networkObjectSchema);
+  //   ajv.addSchema(contractObjectSchema);
+  //   if (ajv.validate("contract-object.spec.json", contractObj)) {
+  //     return contractObj;
+  //   } else {
+  //     console.log("Err", ajv.errors)
+  //     throw ajv.errors;
+  //   }
+  // },
 
```

Comment:
I commented this because I think ti is never invoked. I will delete it in the next PR if I still don't find any usage.